app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,22 @@
  *= require_tree .
  *= require_self
  */
+
+table {
+  border-collapse: separate;
+  border-spacing: 10px 0;
+}
+
+td {
+  padding: 10px 0;
+}
+
+.top-buffer { margin-top:20px; }
+
+hr {
+    border: none;
+    height: 1px;
+    /* Set the hr color */
+    color: #333; /* old IE */
+    background-color: black; /* Modern Browsers */
+}
```

Comment:
You shouldnt write css in application.css, this file should be only to include other files. Write it in an extra file.